README.md

```
@@ -17,11 +17,14 @@ console.log(runtime); // "" | "node" | "deno" | "bun" | "workerd" ...
 console.log(runtimeInfo); // { name: "node" }
 ```
 
-Individual named exports: `isNode`, `isBun`, `isDeno`, `isNetlify`, `isEdgeLight`, `isWorkerd`, `isFastly`
+Individual named exports: `isNode`, `isBun`, `isDeno`, `isNetlify`, `isEdgeLight`, `isWorkerd`, `isFastly`, `isNub`
 
 > [!NOTE]
 > `isNode` is also `true` in Bun/Deno with Node.js compatibility mode. Use `runtime === "node"` for strict checks.
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)